source/_integrations/onewire.markdown

```diff
@@ -12,53 +12,56 @@ ha_codeowners:
 
-Supported devices:
+Each and every 1-wire device has a (globaly) unique ID that identifies the deivce on the bus. The first two digits identifies a device family and the last 14 digita are a globally unique number given to it during manufacturing. 
 
-- [DS18B20](https://datasheets.maximintegrated.com/en/ds/DS18B20.pdf)
-- [DS18S20](https://www.maximintegrated.com/en/products/sensors/DS18S20.html)
-- [DS1822](https://datasheets.maximintegrated.com/en/ds/DS1822.pdf)
-- [DS1825](https://datasheets.maximintegrated.com/en/ds/DS1825.pdf)
-- [DS28EA00](https://datasheets.maximintegrated.com/en/ds/DS28EA00.pdf) temperature sensors
-- [DS2406/TAI-8570](https://datasheets.maximintegrated.com/en/ds/DS2406.pdf) Temperature and pressure sensor made by AAG
-- [DS2438/B1-R1-A](https://datasheets.maximintegrated.com/en/ds/DS2438.pdf) Temperature, pressure and humidity sensor by AAG
+The different families have different functionality and can measure different quantities.
 
-The 1-Wire bus can be connected directly to the IO pins of Raspberry Pi or using dedicated interface adapter (e.g [DS9490R](https://datasheets.maximintegrated.com/en/ds/DS9490-DS9490R.pdf)).
+### Supported devices:
 
-## Raspberry Pi setup
+| Family | Device           | Physical Quantity  |
+| -------|:-----|:-----|
+| 10     | [DS18S20](https://www.maximintegrated.com/en/products/sensors/DS18S20.html)  | Temperature                     |
+| 12     | [DS2406(TAI-8570)](https://datasheets.maximintegrated.com/en/ds/DS2406.pdf)  | Temperature (pressure when using TAI-8570[1]) |
+| 1D     | [DS2423](https://datasheets.maximintegrated.com/en/ds/DS2423.pdf)            | Counter                         |
+| 22     | [DS1822](https://datasheets.maximintegrated.com/en/ds/DS1822.pdf)            |                                 |
+| 26     | [DS2438/B1-R1-A/S2-R1-A [2]](https://datasheets.maximintegrated.com/en/ds/DS2438.pdf)| Temperature, Voltage, Current, (pressure and humidity when using TAI-8570[1]) | 
+| 28     | [DS18B20](https://datasheets.maximintegrated.com/en/ds/DS18B20.pdf)          | Temperature                     |
+| 3B     | [DS1825](https://datasheets.maximintegrated.com/en/ds/DS1825.pdf)            | Temperature                     |
+| 42     | [DS28EA00](https://datasheets.maximintegrated.com/en/ds/DS28EA00.pdf)        | Temperature                     |
+| EF     | [HobbyBoard [3]](https://hobbyboards.com/)                                   | Temperature, Humidity, Moisture |  
 
-In order to setup 1-Wire support on Raspberry Pi, you'll need to edit `/boot/config.txt` following [this documentation](https://www.waveshare.com/wiki/Raspberry_Pi_Tutorial_Series:_1-Wire_DS18B20_Sensor#Enable_1-Wire).
-To edit `/boot/config.txt` on the Home Assistant Operating System, use [this documentation](https://developers.home-assistant.io/docs/en/hassio_debugging.html) to enable SSH and edit `/mnt/boot/config.txt` via `vi`.
+#### Notes:
```

Comment:
```suggestion
## Notes

```

Reply:
My intentions were internal references. [2] refers to [2] in table. Then they might not be rmoved. 
To note where my explanations were located I used ####Notes becaused it looked good. Its not a heading, so #### might not be good. How about making it bold or italic instead? 
OK to keep reference notations ?

Reply:
`####` = a heading, it is a `h4`.

Our website does not support Markdown references or footnotes, unfortunately.